network-api/app/networkapi/news/serializers.py

```diff
@@ -1,9 +1,33 @@
 from rest_framework import serializers
 
+from mezzanine.conf import settings
 from networkapi.news.models import News
 
-
 class NewsSerializer(serializers.ModelSerializer):
+
+    glyph = serializers.SerializerMethodField()
+
+    def get_glyph(self, instance):
+        print (settings.USE_S3)
+
+        # Remote hosted? Return the remote URL
+        if settings.USE_S3:
+            return "{host}{glyph}".format(
+                host=settings.MEDIA_URL,
+                glyph=instance.glyph
+            )
+
+        # Stored on the local filesystem: return as an absolute URL
+        request = self.context['request']
+
+        return "{protocol}://{root}{url}{glyph}".format(
+            protocol='https' if request.is_secure() else 'http',
+            root=request.META['HTTP_HOST'],
+            url=settings.MEDIA_URL,
+            glyph=str(instance.glyph).replace(' ','%20')
+        )
+
```

Comment:
i think it's 1 blank line for nested classes. But personally don't mind as long as flake8 doesn't complain